tests/PhpGenerator/Helpers.dump().phpt

```diff
@@ -39,6 +39,12 @@ Assert::same('[]', Helpers::dump([]));
 Assert::same('[$s]', Helpers::dump([new PhpLiteral('$s')]));
 
 Assert::same('[1, 2, 3]', Helpers::dump([1, 2, 3]));
+Assert::same('[1, 2, 3]', Helpers::dump([1, 2, 3], 91));
+same("[
+	1,
+	2,
+	3,
+]", Helpers::dump([1, 2, 3], 92));
```

Comment:
Length of the `[1, 2, 3]` is *9* + $initLength = *92* is *101* in total and this should not fit into the one line, because the default value of the `Helpers::$WRAP_LENGTH` is 100